src/types.ts

```diff
@@ -1,11 +1,37 @@
-export { Request as ExpressRequest } from 'express';
+type Timestamp = number;
 
-export enum Products {
-  jira = 'jira',
-  confluence = 'confluence',
-  bitbucket = 'bitbucket',
+export const ContextQsh = 'context-qsh';
```

Comment:
Since it's a constant should it be called `CONTEXT_QSH_HEADER`? Or Javascript code style for constants differs from Java world?

Reply:
Not sure to be honest but I personally dislike ALL_UPPERCASE_CONSTANTS and I avoid them in JS/TS as much as possible.